### PR TITLE
ClientAI - Wrapped All functions for the communication between IA and Server, close #22

### LINKED ISSUE #22: ClientAI - Wrapped functions
Wrapped all the required functions to be able to communicate with the C interface
The functions will be given in a shared library, each function will be find with dlsym

- [ ] Forward (move up by one tile) 
- [ ] Right (turn 90° right)
- [ ] Left (turn 90° left)
- [ ] Look (look around)
- [ ] Inventory (check inventory)
- [ ] Broadcast text (broadcast text to others players)
- [ ] Connect_nbr (number of team's unused slot)
- [ ] Fork (fork a player)
- [ ] Eject (Eject players from this tile)
- [ ] Take object (Take an object on the current tile)
- [ ] Set object down (drop an object on the current tile)
- [ ] Incantation (start the incantation to level up)

Please refer to this link if there is any doubt: https://www.notion.so/RFC-ClientIA-Server-0521a9bf917843d09158d8f851ce0d88

### PR DESCRIPTION
**Pull request by 31Nathan**
The client has now two wrappers:
One for the DLLib (DLLibWrapper)
Another one for the Interface between the client and the server (ServerWrapper)
Both are useful to cast python variables in c variables

### CODE REVIEW

File: src/zappy_ai/ai_function_wrapper.py [hunk 1]
```diff
@@ -0,0 +1,280 @@
...
+            self.__GetRepBroadcastText = self.DLLibWrapper.getFunctionFromLibrary("c_interface_get_broadcast_response")
+            self.__GetRepConnectNbr = self.DLLibWrapper.getFunctionFromLibrary("c_interface_get_connectable_response")
+            self.__GetRepFork = self.DLLibWrapper.getFunctionFromLibrary("c_interface_get_fork_response")
+            self.__GetRepEject = self.DLLibWrapper.getFunctionFromLibrary("c_interface_get_eject_response")
+            self.__GetRepTakeObject = self.DLLibWrapper.getFunctionFromLibrary("c_interface_get_take_response")
+            self.__GetRepPlaceObject = self.DLLibWrapper.getFunctionFromLibrary("c_interface_get_place_response")
+            self.__GetRepIncantation = self.DLLibWrapper.getFunctionFromLibrary("c_interface_get_incantation_response")
+        except:
+            return False
+        return True
```
[the remark this answers is not in the document]
Comment (GuyomT) [reply src/zappy_ai/ai_function_wrapper.py:91]: why two underscore ?

[the remark this answers is not in the document]
Comment (31Nathan) [reply src/zappy_ai/ai_function_wrapper.py]: Log Done
Two underscores reference to private members in Python

**Review by Cavonstavant — APPROVED** [review]

File: src/zappy_ai/ai_DLLib_wrapper.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
+import ctypes
+import os
+from _ctypes import dlopen
+from _ctypes import dlsym
+from _ctypes import dlclose
+
+"""
+The libToto
```
Comment (pariciadra) [inline src/zappy_ai/ai_DLLib_wrapper.py:8]: toto lib ? can you maybe choose useful name

**Review by pariciadra — APPROVED** [review]
Change the name of the lib, more accurate could be good, otherwise LGTM

**Pull request merged**